test/integration/prerender/test/index.test.js

```diff
@@ -315,6 +315,11 @@ const runTests = (dev = false, looseMode = false) => {
     expect(data.pageProps.params).toEqual({})
   })
 
+  it('should not error at the root with no trailing slash', async () => {
```

Comment:
This test doesn't fail without the above change due to this not causing `req.url` to be empty